### PR TITLE
Social Icons and Events.jsx

### LINKED ISSUE #114: Fix Social Media Icons
image

- In mobile, the social media icons are smushed together.
- Ideally we want the icons to be spaced out a bit more.
- The name of the social media links should also be clickable.

### LINKED ISSUE #116: Implement Event.jsx
image

(I'll let you know once i setup acm-ucr/calendar)
- There should be a component created for you already
- Only change is to add text which will be the title of the component.
- Pass in color prop.
- Corners should be round but the event should be more square-like compared to original image above.
- After a certain amount of length, the title should cut off (e.g. "The quick brown...")

### PR DESCRIPTION
**Pull request by RussellVillanueva**
Issue fixes

### CODE REVIEW

**Comment by RussellVillanueva** [comment]
The issues fixes is the old one, The fix calendar event size is the correct pr 
image
image

**Comment by RussellVillanueva** [comment]
I put both of my issues in one branch and I selected Fix Social Media Icons as my Development Branch

File: src/components/events/CustomEvents.jsx [hunk 1]
```diff
@@ -1,7 +1,32 @@
+import React from "react";
+
+const categoryColors = {
+  "H4B Food Tabling": "#FFDDE7", // pink
```
Comment (seanquiambao) [inline src/components/events/CustomEvents.jsx]: put these colors in tailwind config

File: src/components/events/CustomEvents.jsx [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
+  };
+
+  const backgroundColor = getColor();
+
   return (
-    <div>
-      <p className="text-black">{event.title}</p>
+    <div
+      className="border rounded-lg p-2 mb-2"
+      style={{ backgroundColor, borderColor: backgroundColor }}
```
Comment (seanquiambao) [inline src/components/events/CustomEvents.jsx]: embed the colors into className

File: src/components/support/Support.jsx [hunk 3]
```diff
@@ -13,9 +14,17 @@ const Support = () => {
...
+              <Subtitle text="Donations" color="text-biscuits-brown-200" />
+            </div>
+          </Link>
           <Button text="VENMO" link="https://account.venmo.com/u/h4b_ucr" />
-          <Subtitle color="text-biscuits-brown-200" text="Newsletter" />
+          <Link href="http://eepurl.com/iaWAc5">
+            <div className="cursor-pointer text-biscuits-brown-200 hover:underline">
+              <Subtitle text="Newsletter" color="text-biscuits-brown-200" />
+            </div>
+          </Link>
```
Comment (seanquiambao) [inline src/components/support/Support.jsx]: revert to old code, this should only apply to the icons

File: src/components/support/Connect.jsx [hunk 4]
```diff
@@ -1,10 +1,16 @@
 import Link from "next/link";
+
 const Connect = ({ name, icon, link }) => {
   return (
-    <div className="flex flex-col items-center text-8xl text-biscuits-purple-200">
-      <Link href={link}>{icon}</Link>
+    <Link
+      href={link}
+      className="flex flex-col items-center text-8xl text-biscuits-purple-200 hover:underline"
```
Comment (seanquiambao) [inline src/components/support/Connect.jsx]: remove underline

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Comment by seanquiambao** [comment]
link the other issue, i think it got removed

File: src/components/support/Connect.jsx (hunk 4 above)
Comment (seanquiambao) [inline src/components/support/Connect.jsx]:
```suggestion
      className="flex flex-col items-center text-8xl text-biscuits-purple-200"
```

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**